includes/rest/class-followers.php

```diff
@@ -5,6 +5,7 @@
 use stdClass;
 use WP_REST_Server;
 use WP_REST_Response;
+use Activitypub\Collection\Users as User_Collection;
```

Comment:
Small nitpick but we should probabl normalize to `UserCollection` or change the below to `Follower_Collection`?